AIRobot.md

```diff
@@ -0,0 +1 @@
+You've been thinking of adopting one of those new AI robofriends. Yes, that's right. I know you've been lonely of late. [Here's an early present then.](nagma-venture/AIattacks.md)
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.